mycroft_admin_service.py

```diff
@@ -124,4 +124,8 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    # Run loop trying to reconnect if there are any issues starting
+    # the websocket
+    while True:
+        main()
```

Comment:
In case the client throws an exception for instance, because the socket is already open, I think we should wrap this in:
```Python
import traceback
try:
    main()
except KeyboardInterrupt:
    raise
except:
    traceback.print_exc()
```